### ISSUE TITLE
[Bug Report] <Talent Page Anchor Issues>

### ISSUE THREAD

**Issue by onyxhell** [Kiatra/BlizzMove#136]
### What game version did the bug occur in?

Retail

### Troubleshooting

- [X] I have verified that BlizzMove is up to date.
- [X] I have tried to reproduce the bug with no other addons loaded.
- [X] I have verified that the issue hasn't yet been reported.

### Describe the bug.

Talent page after being resized or moved out of the way, would anchor to the edge of screen when closed and reopened. I would need to /reload to get it to anchor at the center of screen again. 

### Steps to reproduce.

Open talent page. Move frame anywhere. Close it, attempt to reopen and it would be anchored randomly on edge of screen.

### Expected behavior.

I expected the talent page to stay anchored to the last position I manually put it in.

**Comment by Numynum** [comment]
do you have the setting enabled for remembering frame positions permanently?

I'm also not really sure what you mean frankly, I can move the talent UI, close it and reopen it, and it's exactly where I left it
maybe some screenshots help explain what you mean?

**Comment by onyxhell** [comment]
Yes I do. When I get home I’ll do a screen recording to better explain. On Sep 30, 2024, at 10:36 AM, MarkW ***@***.***> wrote:﻿
do you have the setting enabled for remembering frame positions permanently?
I'm also not really sure what you mean frankly, I can move the talent UI, close it and reopen it, and it's exactly where I left it
maybe some screenshots help explain what you mean?

**Comment by wdeasley** [comment]
I've been experiencing this issue as well.  I see this is 2 weeks old and it's not closed.  I see that there was a request for screen shots so I hope this can help.

Starting from fresh load:
![image](https://github.com/user-attachments/assets/fab7b8fa-01cc-4286-bc3c-535be99cb762)

Moving the frame down:
![image](https://github.com/user-attachments/assets/a5e36a5a-1ec8-435f-a173-083c0d0ea386)

Closing the frame (n) and then re-opening the frame:
![image](https://github.com/user-attachments/assets/e5217087-1e17-40f9-b506-88fb2856905c)

Repeating the close and open action:
![image](https://github.com/user-attachments/assets/e049afa1-d9ad-46dc-9555-236843c06c4d)

again...
![image](https://github.com/user-attachments/assets/4c193d0a-9108-4c30-a69a-9e31d3f761db)

Eventually it will end up off the screen with a corner or sliver of the frame available to drag back.

observation:
Its looking as though it's trying to open based on mouse position.  Once it has reached the bottom of the screen it depends on where the mouse is that it opens far left or right.  (Much harder to capture this behavior in screen shots)

**Comment by Numynum** [comment]
Does that also happen with 0 addons enabled other than blizzmove?
there are a lot of addons that mess around with frame positions (some may do so without you realizing it), and many of them don't work too well, or don't work well if there's any similar addon enabled

**Comment by wdeasley** [comment]
No, Singled out blizzMove and it's not repeating.

I can go through and enable them 1 by 1, would that information be useful, if so I can do so

**Comment by Numynum** [comment]
Use https://curseforge.com/wow/addons/no-auto-close instead of demodal, No Auto Close is designed specifically to be used with blizzmove 🙂

**Comment by Numynum** [comment]
@onyxhell does that solve it for you too?

**Comment by wdeasley** [comment]
re-enabled the rest of my addons, and found that it's still happening, however it appears to be tied to: https://www.curseforge.com/wow/addons/elvui-eltruism

so going to reach out to that community to figure it out

**Comment by Numynum** [comment]
as a tip, you can also use https://www.curseforge.com/wow/addons/bisector to more efficiently find the cause of issues (if I understand it correctly, you'd use `/bisect start +BlizzMove` in step 1)

**Comment by onyxhell** [comment]
Same exact problem as he had. I just uninstalled blizzmove. Was an easier fix for me than to diagnose this issue and debug it. Didn’t have the issue once I disabled blizzmove. On Oct 14, 2024, at 1:09 PM, MarkW ***@***.***> wrote:﻿
as a tip, you can also use https://www.curseforge.com/wow/addons/bisector to more efficiently find the cause of issues (if I understand it correctly, you'd use /bisect start +BlizzMove in step 1)
